R/stabselgraphical.R

```diff
@@ -0,0 +1,67 @@
+#' Create a regularization path - copied from pulsar.
```

Comment:
You can just leave the roxygen markup in place and never run roxygen. I've committed the documentation files that were created so you can ignore it for now, assuming that you intend to go that direction in the future.